src/Designer/frontend/resourceadm/components/AccessListEnvLinks/AccessListEnvLinks.tsx

```diff
@@ -29,13 +30,13 @@ export const AccessListEnvLinks = (): React.JSX.Element => {
   return (
     <div className={classes.envButtonsWrapper}>
       {isLoadingPublishStatus && (
-        <StudioSpinner showSpinnerTitle spinnerTitle={t('resourceadm.loading_publish_status')} />
+        <StudioSpinner aria-label={t('resourceadm.loading_publish_status')} />
       )}
       {publishStatusData && (
         <>
           {envPublishStatus.some((env) => !env.isResourcePublished) && (
-            <StudioAlert severity='warning'>
-              <StudioParagraph size='sm' spacing>
+            <StudioAlert data-color='warning'>
+              <StudioParagraph data-size='sm' spacing>
```

Comment:
`data-size='sm'` is set in `studio-root`, and should propagate downwards, so it might be redundant to specify it. I'm not 100% sure, so this suggestion should be tested first 🙂 

This applies to multiple locations in this PR.
```suggestion
              <StudioParagraph spacing>
```

Reply:
Good points! I will remove `data-size='sm'`, since after testing without it, the UI looks like it should 😀